cli/stubs/magento2/env.php

```diff
@@ -133,4 +133,8 @@
       'ssl' => '',
     ),
   ),
+  'directories' => 
+  array (
+    'document_root_is_pub' => true,
```

Comment:
What is this used for? I searched the repo and couldn't find any other references to it.

Reply:
Hi @lazyguru, good question, this is needed to tell Magento that it is being served out of the pub dir, versus the magento root. When creating files, such as sitemap.xml, this config is polled to determine what https://example.com/sitemap.xml translates to. `/sitemap.xml` or `/pub/sitemap.xml`.

https://devdocs.magento.com/guides/v2.3/install-gde/tutorials/change-docroot-to-pub.html#3-update-the-envphp-file descibes this functionality in-depth.